.template/variants/web/Gemfile.rb

```diff
@@ -23,10 +23,11 @@
   EOT
 end
 
-insert_into_file 'Gemfile', after: %r{gem 'pronto-reek'.*\n} do
+insert_into_file 'Gemfile', after: %r{gem 'danger'.*\n} do
   <<-EOT
-  gem 'pronto-scss', require: false
-  gem 'pronto-eslint_npm', require: false
+  gem 'danger-slim_lint' # Lint Slim template files.
+  gem 'danger-eslint' # Eslint
```

Comment:
```suggestion
  gem 'danger-eslint' # ESLint
```
IMO `ESLint` would be better here.
but anyway, I think a comment is not necessary.
What do you think?

Reply:
Yes we should adapt the better one! Will change soon.

Reply:
@namtx either way we can do it. Having a small description is good, so I guess we can put some more words in it, so that it will be better understandable, what do you think?